src/django_idom/utils.py

```diff
@@ -102,11 +113,14 @@ def _get_components(self, templates: set[str]) -> set[str]:
         for template in templates:
             with contextlib.suppress(Exception):
                 with open(template, "r", encoding="utf-8") as template_file:
-                    match = COMPONENT_REGEX.findall(template_file.read())
-                    if not match:
+                    regex_iterable = COMPONENT_REGEX.finditer(template_file.read())
+                    if not regex_iterable:
                         continue
```

Comment:
This is always going to be true. If the iterable is empty `update()` will do nothing anyway.
```suggestion
```